nautobot/core/templates/inc/table.html

```diff
@@ -1,38 +1,10 @@
-<<<<<<< HEAD
-{% load django_tables2 %}
+{% load querystring from django_tables2 %}
 {% load get_item table_config_button from helpers %}
 
 {% with last_column_name=table.columns.names|last %}
     <table{% if table.attrs %} {{ table.attrs.as_html }}{% endif %}>
         {% if table.show_header %}
             <thead>
-=======
```

Comment:
I am not following what happens here, feel free to resolve though if this makes sense, just trying to call attention to what I can't really review.